src/BSpline.jl

```diff
@@ -255,6 +255,47 @@ function curvepoint(bspline::BSpline, u)
 end
 
 
+"""
```

Comment:
Let's either put this in the NURBS.jl file, or create a utilities file to put this in.  Since it's general, I don't think the BSpline.jl file is the best location, just for organization purposes.